problem with angular 4.3x... Cannot find module 'ngx-ui-switch'

hi,

I'm getting the below error after npm i


ERROR in D:/PROJELER/2017/tps-hr/web/src/app/system/personnel/personnel.module.ts (7,32): Cannot find module 'ngx-ui-switch'.
ERROR in Error: Error encountered resolving symbol values statically. Could not resolve ngx-ui-switch relative to D:/PROJELER/2017/tps-hr/web/s
ymbol PersonnelModule in D:/PROJELER/2017/tps-hr/web/src/app/system/personnel/personnel.module.ts, resolving symbol PersonnelModule in D:/PROJE
dule.ts
    at syntaxError (D:\PROJELER\2017\tps-hr\web\node_modules\@angular\compiler\bundles\compiler.umd.js:1729:34)
    at simplifyInContext (D:\PROJELER\2017\tps-hr\web\node_modules\@angular\compiler\bundles\compiler.umd.js:25118:23)
    at StaticReflector.simplify (D:\PROJELER\2017\tps-hr\web\node_modules\@angular\compiler\bundles\compiler.umd.js:25130:13)
    at StaticReflector.annotations (D:\PROJELER\2017\tps-hr\web\node_modules\@angular\compiler\bundles\compiler.umd.js:24558:41)
    at _getNgModuleMetadata (D:\PROJELER\2017\tps-hr\web\node_modules\@angular\compiler-cli\src\ngtools_impl.js:138:31)
    at _extractLazyRoutesFromStaticModule (D:\PROJELER\2017\tps-hr\web\node_modules\@angular\compiler-cli\src\ngtools_impl.js:109:26)
    at D:\PROJELER\2017\tps-hr\web\node_modules\@angular\compiler-cli\src\ngtools_impl.js:129:27
    at Array.reduce (native)
    at _extractLazyRoutesFromStaticModule (D:\PROJELER\2017\tps-hr\web\node_modules\@angular\compiler-cli\src\ngtools_impl.js:128:10)
    at D:\PROJELER\2017\tps-hr\web\node_modules\@angular\compiler-cli\src\ngtools_impl.js:129:27
    at Array.reduce (native)
    at _extractLazyRoutesFromStaticModule (D:\PROJELER\2017\tps-hr\web\node_modules\@angular\compiler-cli\src\ngtools_impl.js:128:10)
    at Object.listLazyRoutesOfModule (D:\PROJELER\2017\tps-hr\web\node_modules\@angular\compiler-cli\src\ngtools_impl.js:53:22)
    at Function.NgTools_InternalApi_NG_2.listLazyRoutes (D:\PROJELER\2017\tps-hr\web\node_modules\@angular\compiler-cli\src\ngtools_api.js:91:3
    at AotPlugin._getLazyRoutesFromNgtools (D:\PROJELER\2017\tps-hr\web\node_modules\@ngtools\webpack\src\plugin.js:207:44)
    at _donePromise.Promise.resolve.then.then.then.then.then (D:\PROJELER\2017\tps-hr\web\node_modules\@ngtools\webpack\src\plugin.js:443:24)
    at process._tickCallback (internal/process/next_tick.js:109:7)

--- a/build.ts
+++ b/build.ts
@@ -14,6 +14,7 @@ const pkg = require(`${process.cwd()}/package.json`);
 const GLOBALS = {
   '@angular/core': 'ng.core',
   '@angular/common': 'ng.common',
+  '@angular/forms': 'ng.forms',
   '@angular/animations': 'ng.animations',
   '@angular/platform-browser': 'ng.platformBrowser',
   'rxjs': 'Rx',

--- a/package.json
+++ b/package.json
@@ -21,7 +21,7 @@
     "start": "ng serve",
     "build": "rimraf dist && ts-node build.ts",
     "build:demo": "rimraf demo && ng build --base-href=/ngx-ui-switch/demo/ --output-path=demo",
-    "publish": "cd dist/packages-dist && npm publish",
+    "publish": "cd dist/packages-dist && yarn publish",
     "bundlesize": "bundlesize",
     "test": "ng test",
     "test:ci": "ng test --browser ChromeHeadless --watch false",

--- a/package.json
+++ b/package.json
@@ -1,6 +1,6 @@
 {
   "name": "ngx-ui-switch",
-  "version": "1.4.0",
+  "version": "1.4.4",
   "description": "switch button for angular4",
   "bugs": "https://github.com/webcat12345/ngx-ui-switch/issues",
   "homepage": "https://github.com/webcat12345/ngx-ui-switch#readme",

--- a/src/lib/package.json
+++ b/src/lib/package.json
@@ -1,6 +1,6 @@
 {
   "name": "ngx-ui-switch",
-  "version": "1.3.9",
+  "version": "1.4.4",
   "description": "switch button for angular4",
   "bugs": "https://github.com/webcat12345/ngx-ui-switch/issues",
   "homepage": "https://github.com/webcat12345/ngx-ui-switch#readme",

